godtools/Views/TractElements/TractTextContentProperties.swift

```diff
@@ -15,11 +15,11 @@ class TractTextContentProperties: TractProperties {
     var i18nId: String = ""
     var textAlign: NSTextAlignment = .left
     // textColor
-    var textScale: CGFloat = 1.0
+    // textScale
     var value: String = ""
     
     override func defineProperties() {
-        self.properties = ["i18nId", "textScale", "value"]
```

Comment:
@pablomarti  so does this mean we can't scale individual text elements any more? we need to be able to do that.